examples/cloud_run_cloudevents/README.md

```diff
@@ -0,0 +1,23 @@
+# Deploying a CloudEvent Function to Cloud Run with the Functions Framework
+
+This sample uses the [CloudEvents SDK](https://github.com/cloudevents/sdk-python) to send and receive a CloudEvent on Cloud Run.
+
+## How to run this locally
+
+Build the Docker image:
+
+```commandline
+docker build -t ff_example .
+```
+
+Run the image and bind the correct ports:
+
+```commandline
+docker run --rm -p 8080:8080 -e PORT=8080 ff_example
+```
+
+Send an event to the container:
+
+```python
+python send_cloudevent.py
```

Comment:
Would be nice to execute this inside the container instead, with something like `docker run -t ff_example send_cloudevent.py`. This ensures that a) the user doesn't need the right version of Python installed locally to run it and b) any networking is guaranteed to work. We'd need to make this script executable and have a Python shebang as well.